plugins/foreman_remote_execution/nightly/index.md

```diff
@@ -0,0 +1,585 @@
+
+![Job Invocation schedule form](/plugins/foreman_remote_execution/{{page.version}}/invocation_form_scheduling.png)
+
+While execute now is obvious, future time scheduling offers you to customize
+two things. First is when the job should start, the second one called **Start
+before** determines the deadline after which the execution shouldn't start.
+This can be useful when you have some maintenance window and if the execution
+gets delayed it wouldn't be safe to run when the window was closed.
+ 
+When settings up recurring execution there are many ways how you can specify
+the interval. You can choose from hourly, daily, weekly, monthly interval
+basis and for advanced users there's option to specify custom cron
+configuration. This way you can setup various intervals, for example
+
+    # Run on every second Sunday of every month
+    0 4 8-14 * *
+
+    # Run every 5 minutes during 6 AM and 6 PM
+    */5 6,18 * * *
+
+You can specify the number of runs as **Repeat N times** attribute or limit
+it by **Ends at** date. What happens sooner takes precedence. You can keep
+the invocation configured without any fixed ending condition. If you want
+to stop such job later, you either cancel the job and cancel the recurring
+logic.
+
+After submitting the form, you're redirected to the Job detail which informs
+you about the progress.
+
+## 3.3 Job detail
+
+On the Job detail page you can see overall progress and result of the job
+execution.
+
+![Job detail 1](/plugins/foreman_remote_execution/{{page.version}}/job_detail_1.png)
+
+In the overview tab you can see what query was used to find the hosts and how
+many hosts were found using it. Below is the evaluated template in generic
+form. If there were some fact or parameter inputs used, you will see only
+placeholders for them since they are specific to each host. User inputs are
+evaluated since they are same for all hosts. A chart is showing the status of
+the job on all targets.
+
+From this screen you can even cancel the job if it's not finished yet.  First
+cancel attempt will try to cancel the run on the Smart Proxy. If proxy does not
+respond, you can hit the cancel button a second time which forces the Job to be
+cancelled on Foreman (while it still might be running on Smart Proxy). Last Job
+Task takes you to Foreman Tasks that represents this Job.
+
+In Hosts tab you can see all hosts targeted by this execution, status
+of the job on them and link to Foreman Task that represent the
+execution for a particular host. Note that this task is a subtask of a
+whole Job task.
+
+If you scheduled a recurring you'll also see **Recurring logic** tab. This
+gives you some basic information, e.g. when next run is supposed to happen.
+After you cancel the job, it cancels all future executions as well.
+
+## 3.4 Jobs list
+
+You can find all Jobs when you navigate to Monitor > Jobs. A table
+lists all jobs from history to future. You can search jobs by **Job name** with
```

Comment:
There will be more occurrences, I'll go through the whole manual once the PR is merged and send a separate PR.